docs/introduction.rst

```diff
@@ -11,106 +11,143 @@ or modifying files produced by other programs (e.g.,
+older nodes to the current generation; any historical nodes whose genetic contribution to
+the samples has been whittled down to zero are also removed. Finally, note that to
+reconstruct relationships between samples, we strictly only need to keep a node if it
+represents the genetic *most recent common ancestor* (MRCA) of at least two samples. So
+by default, we also remove historical nodes that are only "on the way to" a sample, but
+do not represent a split in the tree (to alter this behaviour, check out the
+``retainCoalescentOnly`` parameter of ``treeSequenceInitialize()``)
+
+*****************************************************
+What if I'm interested in historical individuals too?
+*****************************************************
+
+As we've seen, the tree sequence retains enough of the nodes (genomes) in the past to
+reconstruct the genetic ancestry of the currently alive individuals. But you might want
+more than that. For example, there may be individuals who are not alive any more, but
+whose complete ancestry you would like to know.  Or it may be that you want to access the
+spatial location of genomes in the past (which, for historical reasons is linked to
+individuals, not to genomes). This is done by *remembering* an individual during the
+simulation, using the SLiM function ``treeSeqRememberIndividuals()``. 
+
+.. _sec_remembering_individuals:
+
+By default, individuals are *permanently* remembered, by marking their nodes as actual
+samples: the simulated equivalent of ancient DNA dug out of permafrost, or stored in an
+old collecting tube. This means any tree sequence subsequently recorded will always
+contain this individual, its nodes (now marked as samples), and its full ancestry. 
+
+.. _sec_retaining_individuals:
+
+Alternatively, you may want to avoid treating historical individuals and their genomes as
+actual samples, but simply *retain* them as long as they are relevant to the genetic
+ancestry of the sample nodes. You can do this by using
+``treeSeqRememberIndividuals(..., permanent=F)``. In this case, the individual's nodes
+are not marked as samples, and are therefore subject to the normal removal rules. If only
+one is removed, you may, unusually, end up with a individual containing only one genome.
+However, as soon as both nodes of a retained individual have been lost, the individual
+itself is deleted too.
+
+These following diagram adds these two possibilities to the example above, with a
+reminder of the terminology we have now introduced.
 
 .. figure:: _static/pedigree3.png
    :scale: 40%
 
-.. note::
-
-   In previous versions of SLiM/pyslim, the *first generation* of individuals
-   were kept in the tree sequence, to allow recapitation.
-   This is no longer necessary, so these are *no longer present*, unless
-   you specifically Remember them -- see the :ref:`sec_tutorial_recapitation`
-   section of the tutorial for more discussion of how this works.
```

Comment:
I think we could leave this note in for the time being?

Reply:
Oh yes, I forgot about putting that back in, sorry. I'll find a place for it. If we move the "historical individuals" bit to the tutorial (which I'm not averse to), then perhaps that's a good place to put it, so it's not so in-your-face in the introduction.